app/code/Magento/Tax/Test/Unit/Plugin/Checkout/CustomerData/CartTest.php

```diff
@@ -77,49 +79,34 @@ public function testAfterGetSectionData()
             ]
         ];
 
-        $this->checkoutHelper->expects(
-            $this->atLeastOnce()
-        )->method(
-            'formatPrice'
-        )->willReturn('formatted');
-
-        $item1 = $this->createMock(\Magento\Quote\Model\Quote\Item::class);
-        $item2 = $this->createMock(\Magento\Quote\Model\Quote\Item::class);
-
-        $item1->expects(
-            $this->atLeastOnce()
-        )->method(
-            'getItemId'
-        )->willReturn(1);
-        $item2->expects(
-            $this->atLeastOnce()
-        )->method(
-            'getItemId'
-        )->willReturn(2);
-
-        $this->quote->expects(
-            $this->any()
-        )->method(
-            'getAllVisibleItems'
-        )->willReturn([
-            $item1,
-            $item2,
-        ]);
-
-        $this->itemPriceRenderer->expects(
-            $this->atLeastOnce(1)
-        )->method(
-            'toHtml'
-        )->willReturn(1);
+        $this->checkoutHelper->method('formatPrice')
+            ->willReturn('formatted');
+
+        $item1 = $this->createMock(Item::class);
+        $item2 = $this->createMock(Item::class);
+
+        $item1->method('getItemId')
+            ->willReturn(1);
+        $item2->method('getItemId')
+            ->willReturn(2);
+
+        $this->quote->method('getAllVisibleItems')
+            ->willReturn([
+                $item1,
+                $item2,
+            ]);
+
+        $this->itemPriceRenderer->method('toHtml')
```

Comment:
@joni-jones @buskamuza It looks like missing check that method toHtml will be called at least once

Reply:
@ihor-sviziev, this `expects` is redundant because if `toHtml()` won't be called the test will fail. "Price assertions" check this case.